plugins/Morpheus/stylesheets/main.less

```diff
@@ -466,7 +466,23 @@ div.dataTableVizHtmlTable:not(.dataTableActions), div.dataTableVizAllColumns, di
 }
 
 div.sparkline {
-    display: block;
+    display: -ms-flexbox;
+    -ms-box-orient: horizontal;
+    -ms-box-pack: center;
+
+    display: -webkit-flex;
+    display: -moz-flex;
+    display: -ms-flex;
+    display: flex;
+   
+    -ms-flex: 1 1 auto;
+    flex: 1;
```

Comment:
Since this element is the flex container and not a flex child, these two styles aren't needed. Can you remove them?

Reply:
Add them to the text wrapper instead?